core/src/main/java/org/apache/carbondata/core/metadata/SegmentFileStore.java

```diff
@@ -877,6 +877,10 @@ public SegmentFile getSegmentFile() {
         }
         if (entry.getValue().status.equals(SegmentStatus.SUCCESS.getMessage())) {
           String mergeFileName = entry.getValue().getMergeFileName();
+          // remove file separator if already present
```

Comment:
better to store the correct path while taking input to add segment command itself instead of adding a hack like this?

Reply:
The path stored is always correct. It depends on the user whether he is putting '/' in the end for the segment path or not.

Reply:
if the user is putting "/", while storing itself, can we remove that and store it, instead of skipping here ? because some other place if the stored value is used. again same code has to be there also

Reply:
done